acceptance-tests/src/main/java/io/blueocean/ath/pages/blue/EditorPage.java

```diff
@@ -1,19 +1,19 @@
 package io.blueocean.ath.pages.blue;
 
-import com.google.common.base.Strings;
 import io.blueocean.ath.WaitUtil;
 import io.blueocean.ath.model.MultiBranchPipeline;
-import org.apache.log4j.Logger;
+import org.apache.commons.lang3.StringUtils;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.PageFactory;
 import org.openqa.selenium.support.ui.ExpectedConditions;
-
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
 import javax.inject.Inject;
 
 public class EditorPage {
 
-    private Logger logger = Logger.getLogger(EditorPage.class);
+    private Logger logger = LoggerFactory.getLogger(EditorPage.class);
```

Comment:
The change from log4j to slf4j is not mentioned in the description.
Some people's custom logger configurations might need to be changed because of this, so should be mentioned in the changelog.

Reply:
this is part of BlueOcean acceptance-tests module so it's not a plugin neither executed within Jenkins test framework. 
It's only used here for testing purpose. The idea was to have everything using same logging mechanism as some classes were using slf4j some others not